TradeItIosTicketSDK2/TradeItInstrumentOrderCapabilitiesHelpers.swift

```diff
@@ -31,10 +30,10 @@ extension TradeItInstrumentOrderCapabilities {
         return capabilities.first { $0.value == value }?.value ?? capabilities.first?.value
     }
 
-    func supportedOrderQuantityTypesFor(action: TradeItOrderAction) -> [OrderQuantityType] {
-        let capabilities = capabilitiesFor(field: .actions) as? [TradeItInstrumentActionCapability] ?? []
-        let capability = capabilities.first { $0.value == action.rawValue } ?? capabilities.first
-        return capability?.supportedOrderQuantityTypes.compactMap(OrderQuantityType.init) ?? []
+    func supportedOrderQuantityTypesFor(action: TradeItOrderAction, priceType: TradeItOrderPriceType) -> [OrderQuantityType] {
+        let actionQuantityTypes = supportedOrderQuantityTypesFor(capabilities: self.actions, value: action.rawValue)
+        let priceTypeQuantityTypes = supportedOrderQuantityTypesFor(capabilities: self.priceTypes, value: priceType.rawValue)
+        return actionQuantityTypes.filter(priceTypeQuantityTypes.contains)
```

Comment:
Gets the intersection of the arrays

Reply:
i was a little thrown by the syntax here (are we passing arguments implicitly to `priceTypeQuantityTypes.contains`?). Doesn't seem that the IDE has a clear sense of what's going on as I can't jump to the definition of contains.

Might be clearer to use Swift's Set [`interesect` method](https://stackoverflow.com/a/29852086/3837185)